src/BenchmarksDriver/Program.cs

```diff
@@ -111,6 +111,9 @@ public static int Main(string[] args)
                 "-w|--webHost",
                 "WebHost (Kestrel or HttpSys). Default is Kestrel.",
                 CommandOptionType.SingleValue);
+            var targetFrameworkOption = app.Option("-r|--framework",
```

Comment:
I think all instances of `targetFramework` should be renamed to `framework`, including the SQL column.